packages/graphback/src/database/DatabaseConnectionOptions.ts

```diff
@@ -0,0 +1,5 @@
+export interface DatabaseConnectionOptions {
+  client: string;
+  // tslint:disable-next-line: no-any
+  connectionOptions: any;
```

Comment:
How about our strategies will understand specific implementation (in future) and we will make them explicitly be aware of the engine they work with. Like `PostgressKnexDropAllNoMatterWhatStrategy`. WDYT?